cypress/support/components/TrainDialogsGrid.js

```diff
@@ -33,3 +33,23 @@ export function GetDescription() { return helpers.StringArrayFromElementText('[d
 export function VerifyErrorIconForTrainGridRow(rowIndex) { cy.Get(`div.ms-List-cell[data-list-index="${rowIndex}"]`).find('[data-testid="train-dialogs-validity-indicator"]') }
 
 export function VerifyDescriptionForRow(row, description) { cy.Get(`div[data-item-index=${row}][data-automationid="DetailsRow"]`).find('span[data-testid="train-dialogs-description"]').contains(description) }
+
```

Comment:
The code below was moved here from Train.js since it always should have been in this file.